agent/test-agent-cli/src/terminate.rs

```diff
@@ -0,0 +1,75 @@
+use crate::error::{self, ArchiveSnafu, ClientSnafu, Result};
+use argh::FromArgs;
+use snafu::ResultExt;
+use std::fs;
+use std::fs::File;
+use std::path::PathBuf;
+use std::time::Duration;
+use tar::Builder;
+use test_agent::{Client, DefaultClient};
+use tokio::time::sleep;
+
+#[derive(Debug, FromArgs, PartialEq)]
+#[argh(subcommand, name = "terminate", description = "complete the test")]
+pub(crate) struct Terminate {
+    #[argh(short = 'd', option, description = "results directory")]
+    results_dir: String,
+}
+
+impl Terminate {
+    pub(crate) async fn run(&self, k8s_client: DefaultClient) -> Result<()> {
+        let test_status = k8s_client.test_status().await.context(ClientSnafu)?;
+        if test_status.agent.results.is_empty() {
+            println!("No test results have been sent to Kubernetes cluster.");
+        }
```

Comment:
In my opinion its better to have this, because if there will be any first time user and test status will change to NoTest, they may be wondering what happened.. And this message can help them. Although we can use info instead of print

Reply:
@etungsten Suggested that we can remove this but can be more explicit in test status saying "NoTestResults". Will that be ok to implement?